charts/kube-prometheus-stack/hack/update_mixins.sh

```diff
@@ -2,6 +2,14 @@
 
 set -euo pipefail
 
+sedi () {
```

Comment:
This check is not bullet proof. Some folks, like me 😊, install the GNU version of sed on macOS and set it as the default one.

I would do something like this instead to detect the correct version:
```sh
if [[ "$(sed --version 2>/dev/null)" =~ ^.*GNU.*$ ]]; then
```

Reply:
I added the logic from my helm-secrets project to cover all known cases. 

And I put the condition outside of the function, because than it needs to runs once only.